mycroft/listener/service.py

```diff
@@ -40,10 +40,10 @@ def on_error(e='Unknown'):
     LOG.error('Audio service failed to launch ({}).'.format(repr(e)))
 
 
-class SpeechClient(Thread):
+class SpeechService(Thread):
```

Comment:
This is a breaking change for Neon.. Can we add a backwards-compat import at least until `0.1.0`?